src/gui/mainwindow.cpp

```diff
@@ -213,8 +219,11 @@ MainWindow::MainWindow(IGUIApplication *app, WindowState initialState)
         UIThemeManager::instance()->getIcon(u"folder-remote"_qs),
 #endif
         tr("Transfers"));
-
-    connect(m_searchFilter, &LineEdit::textChanged, m_transferListWidget, &TransferListWidget::applyNameFilter);
+    auto emitFilterChanged = [this](){ emit filterChanged(m_searchFilter->text(),
```

Comment:
Using yet another signal seems to be overkill. Why not just call `applyFilter()` directly?